docs/topics/proc_configuring-jenkins-to-deploy-an-application-with-fmp.adoc

```diff
@@ -0,0 +1,40 @@
+= Configuring Jenkins to deploy your application with the Fabric8 Maven Plugin
+
+Similar to using maven and the Fabric8 Maven Plugin from your local host to deploy an application, you can configure Jenkins to use maven and the Fabric8 Maven Plugin to deploy an application.
+
+[discrete]
+== Prerequisites
+* Access to an OpenShift cluster.
+* Jenkins installed and configured to access the same OpenShift cluster.
+* JDK and Maven installed and configured on your Jenkins server.
+* An application available in GitHub and configured to use maven and the Fabric8 Maven Plugin.
+
+[discrete]
+== Procedure 
+. Create a new OpenShift project for your application.
```

Comment:
Do we expect the user to be able to do this on their own? If so, disregard this comment. If not, maybe a little expansion would be in order.

Reply:
I'm assuming the user can do this, but i'll expand this with sub-steps